docs/SUMMARY.md

```diff
@@ -33,6 +33,7 @@
     * [API Tokens](user-guide/global-configurations/authorization/api-tokens.md)
   * [Notifications](user-guide/global-configurations/manage-notification.md)
   * [External Links](user-guide/global-configurations/external-links.md)
+  * [Catalog Framework](user-guide/global-configurations/catalog-framework.md)
```

Comment:
Not just here, I guess we should also have an entry near Application for application catalog --> Tailored for Adding application related information

Reply:
Yes, we are planning to rewrite all the existing overview pages of Applications, Jobs, Clusters, etc. to accommodate this. But that will be in a separate PR.